bin/docker_push_sha.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+IMG_URL="${SERVER}/${DOCKER_REPOSITORY_NGINX}"
+
+docker tag "${IMG_URL}:${SHA_TAG}" "${IMG_URL}:${REF_TAG}" &&\
+docker push "${IMG_URL}:${SHA_TAG}" &&\
+docker push "${IMG_URL}:${REF_TAG}"
```

Comment:
consider adding empty new line

Reply:
you can build an image with several tag and push without specifying which tag, which will push all tags to the image registry

Reply:
@OriHoch what do you say about that?

Reply:
@idosavion this is the code that works before, maybe it's a good idea to use `BUILDKIT_INLINE_CACHE` but I need to test that before, let's, I think it's a task for new PR.